sdk/tests/platform/test_az_mqtt5_policy.c

```diff
@@ -38,8 +44,11 @@ void az_platform_critical_error(void) { assert_true(false); }
 #define TEST_MQTT_PROPERTY_MESSAGE_EXPIRY_INTERVAL 1000
 #define TEST_MQTT_PROPERTY_PAYLOAD_FORMAT_INDICATOR 0
 #define TEST_MQTT_PROPERTY_CORRELATION_DATA "1234"
+#define TEST_CERTIFICATE_PATH ""
```

Comment:
@danieljurek In order to properly run the integration test to connect to the TLS endpoint, I need to specify the paths to the certificates I created here. The way I did it was to use the `Setup-MQTTContainer.ps1` script to overwrite these three lines (47-49) with the correct path to the certificates but I am getting a style error. 

When I attempt to repro on my end (i.e run the string replacement part of `Setup-MQTTContainer.ps1`) my clang-format does not pick up any errors. I assume then that the issue comes from the fact that the file was changed? Is there a better way to change these #defines?

Reply:
These can't be env vars to avoid modifying the .c file?

Reply:
https://github.com/Azure/azure-sdk-for-c/blob/257597028d540ceaac0106fc34b3fedd4ce57d10/sdk/samples/iot/iot_sample_common.c#L113